department-of-veterans-affairs/abd-vro
# MCP-2246 Reformat date

## What was the problem?

Leading zeros are not consistent with design for PDF, need more custom formatting of dates. (1/1/2023 instead of 01/01/2023)

https://amida.slack.com/archives/C01Q7979Z7D/p1673968482827509?thread_ts=1672172009.573269&cid=C01Q7979Z7D

https://amida.slack.com/archives/C01Q7979Z7D/p1672172009573269

Associated tickets or Slack threads:

- [MCP-2246](https://amida.atlassian.net/browse/MCP-2246)

## How does this fix it?[^1]

Reformat dates to remove leading zeros

## How to test this PR
- Step 1 Run tests, see expected output
- Step 2[^secrel]


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
